<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BGTEST</class>
 <widget class="QMainWindow" name="BGTEST">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1108</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>BGTEST</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>0</y>
      <width>149</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>串口设置</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="4" column="1">
      <widget class="QComboBox" name="StopBits">
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="label_13">
       <property name="text">
        <string>读取进度</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="Baud">
       <item>
        <property name="text">
         <string>9600</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>19200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>38400</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>115200</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>204800</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>波特率</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="DataBits">
       <item>
        <property name="text">
         <string>8</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="Parity">
       <item>
        <property name="text">
         <string>NONE</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="5" column="0" alignment="Qt::AlignHCenter">
      <widget class="QLabel" name="State_com">
       <property name="text">
        <string>未连接</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="Com"/>
     </item>
     <item row="6" column="1">
      <widget class="QSlider" name="timerSlider">
       <property name="minimum">
        <number>0</number>
       </property>
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="singleStep">
        <number>1</number>
       </property>
       <property name="pageStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>2</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>串口号</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>数据位</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>停止位</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>校验位</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_20">
       <property name="text">
        <string>文件速度</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QProgressBar" name="readfile_progressBar">
       <property name="maximum">
        <number>100</number>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QPushButton" name="open_com">
       <property name="text">
        <string>打开串口</string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_24">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>文件(KB)</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="QLabel" name="label_25">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>0</y>
      <width>511</width>
      <height>411</height>
     </rect>
    </property>
    <property name="title">
     <string>原协议数据</string>
    </property>
    <widget class="QTabWidget" name="tabWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>12</y>
       <width>491</width>
       <height>391</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>定位信息</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_5">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>定位信息</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="5" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>地面航向角</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="longitude"/>
          </item>
          <item row="10" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>PDOP</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="gps_num_id"/>
          </item>
          <item row="5" column="1">
           <widget class="QLineEdit" name="course_angle"/>
          </item>
          <item row="10" column="1">
           <widget class="QLineEdit" name="pdop"/>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="speed"/>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="GPS_number_label">
            <property name="text">
             <string>GPS卫星ID</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="time_lineEdit"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>经度</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>纬度</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QLineEdit" name="latitude"/>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>海拔高度</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>北京时间</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="BD_num_id"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>速度</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="height"/>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="BD_number_lable">
            <property name="text">
             <string>BD卫星ID</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QLineEdit" name="positioning_state"/>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>定位状态</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>原始数据</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="0" column="0">
        <widget class="QPlainTextEdit" name="receiveDate"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>XYZ</string>
      </attribute>
      <widget class="QWidget" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>481</width>
         <height>361</height>
        </rect>
       </property>
       <layout class="QGridLayout" name="gridLayout_6">
        <item row="0" column="1">
         <widget class="QLineEdit" name="X"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_22">
          <property name="text">
           <string>Y</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="Y"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_23">
          <property name="text">
           <string>Z</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="Z"/>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label_21">
          <property name="text">
           <string>X</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>定位偏差曲线</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7">
       <item row="0" column="0">
        <widget class="QCustomPlot" name="widget_track" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>412</y>
      <width>641</width>
      <height>381</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="1">
      <widget class="QLabel" name="gps_num_use">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Agency FB&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>210</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="5">
      <widget class="QLabel" name="bd_num_use">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Agency FB&quot;;</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_11">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Agency FB&quot;;</string>
       </property>
       <property name="text">
        <string>GPS</string>
       </property>
      </widget>
     </item>
     <item row="0" column="4">
      <widget class="QLabel" name="label_17">
       <property name="styleSheet">
        <string notr="true">font: 16pt &quot;Agency FB&quot;;</string>
       </property>
       <property name="text">
        <string>BD</string>
       </property>
      </widget>
     </item>
     <item row="0" column="6">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="6">
      <spacer name="verticalSpacer_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="2">
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="0" rowspan="4" colspan="3">
      <widget class="QTableWidget" name="GPS_tableWidget">
       <column>
        <property name="text">
         <string>卫星编号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>仰角</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>方位角</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>信噪比</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="1" column="4" rowspan="4" colspan="5">
      <widget class="QTableWidget" name="BD_tableWidget">
       <property name="columnCount">
        <number>4</number>
       </property>
       <column>
        <property name="text">
         <string>卫星编号</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>仰角</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>方位角</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>信噪比</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>410</y>
      <width>431</width>
      <height>381</height>
     </rect>
    </property>
    <property name="title">
     <string>地图显示</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="1" column="1">
      <widget class="QAxWidget" name="axWidget"/>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="dingwei">
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255, 0, 0, 255), stop:0.166 rgba(255, 255, 0, 255), stop:0.333 rgba(0, 255, 0, 255), stop:0.5 rgba(0, 255, 255, 255), stop:0.666 rgba(0, 0, 255, 255), stop:0.833 rgba(255, 0, 255, 255), stop:1 rgba(255, 0, 0, 255));
font: 10pt &quot;Segoe Print&quot;;</string>
       </property>
       <property name="text">
        <string>定位</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>431</width>
      <height>401</height>
     </rect>
    </property>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>10</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 0, 0);
font: 14pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>BD</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>40</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 119, 255);
font: 14pt &quot;Agency FB&quot;;</string>
     </property>
     <property name="text">
      <string>GPS</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_16">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>401</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_16</zorder>
   <zorder>groupBox</zorder>
   <zorder>groupBox_3</zorder>
   <zorder>widget_2</zorder>
   <zorder>groupBox_4</zorder>
   <zorder>widget</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1108</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_2"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>读取文件回放</string>
   </property>
  </action>
  <action name="action_2">
   <property name="text">
    <string>保存原始文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QAxWidget</class>
   <extends>QWidget</extends>
   <header>qaxwidget.h</header>
  </customwidget>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
